edp.so/CMakeLists.txt

```diff
@@ -78,6 +78,16 @@ add_compile_options(-DBUILD_VERSION_STRING="${BUILD_VERSION_STRING}")
 set(ROCBLAS_LIB "rocblas")
 set(HIP_HCC_LIB "amdhip64")
 
+#ROCBLAS VERSION CHECK FLAGS TO CHECK REORG VERSION 2.44.0
+add_compile_options(-DRVS_ROCBLAS_VERSION_FLAT=${RVS_ROCBLAS_VERSION_FLAT})
+if(RVS_ROCBLAS_VERSION_FLAT LESS 2044000)
+	set(ROCBLAS_MODULE_NM_PREFIX "")
```

Comment:
Isnt is better to add this set in the main CMakefile itself. so that same code repeat in not required in multiple sub cmake?